GDAXSharp/WebSocket/Models/Request/TickerChannel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace GDAXSharp.WebSocket.Models.Request
+{
+    public class TickerChannel
+    {
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("product_ids")]
+        public List<string> ProductIds { get; set; }
```

Comment:
can use the enum here similar to this:

https://github.com/dougdellolio/gdax-csharp/blob/master/GDAXSharp/Services/Orders/Models/Responses/OrderResponse.cs#L18